project-reports-lib.ts

```diff
@@ -75,6 +76,58 @@ export function getStringFromLabel(card: ProjectIssue, re: RegExp): string {
   return str
 }
 
+export function getLastCommentField(
+  issue: ProjectIssue,
+  field: string
+): string {
+  let val = ''
+
+  if (!issue.comments) {
+    return ''
+  }
+
+  for (let i = issue.comments.length - 1; i >= 0; i--) {
```

Comment:
Is the iteration index used elsewhere? Otherwise it may be easier to read with a `for (const issue of issue.comments)` as you do below with `lines`.

Reply:
I'm iterating from last to first ...